src/bazel_version.rs

```diff
@@ -96,12 +97,12 @@ pub fn determine_bazelisk_version(path: &Path) -> Option<String> {
                 return Some(version_str.trim().to_string());
             }
         }
-        None
-    } else if let Ok(bazelversion) = fs::read_to_string(workspace_root.join(".bazelversion")) {
-        return Some(bazelversion.trim().to_string());
-    } else {
-        None
     }
+    if let Ok(bazelversion) = fs::read_to_string(workspace_root.join(".bazelversion")) {
+       return Some(bazelversion.trim().to_string());
```

Comment:
This time, CI is complaining about code formatting / indentation

```suggestion
        return Some(bazelversion.trim().to_string());
```

Reply:
My bad havent worked in Rust in a while, updated!